css/style.css

```diff
@@ -1,9 +1,243 @@
+  text-align: center;
+}
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+header,
+main,
+footer {
+  max-width: 1100px;
+  margin: 0 auto;
+}
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+header {
+  height: 4.3rem;
+  align-items: center;
+  top: 0px;
+  background-color: rgb(248, 248, 248);
+}
+
+.header {
+  height: 4rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  /* background-color: teal;
+  position: fixed; */
+  z-index: 1;
+  margin: 0 7rem;
+  background-color: rgb(248, 248, 248);
+}
+
+.logo-image {
+  height: 40px;
+  width: auto;
+}
+
+.nav label,
+#hamburger {
+  display: none;
+}
+
+.nav-menu {
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  font-size: 1rem;
+  font-weight: 400;
+}
+
+.list-item {
+  margin-left: 25px;
+  padding: 0.6rem 0;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: rgb(148, 143, 143);
+}
+
+.nav-link:hover {
+  color: #e0633a;
+}
+
+.nav-active {
+  color: rgb(75, 73, 73);
+  font-weight: 500;
+}
+
+main {
+  /* margin-top: 5rem; */
+}
+
+.home {
+  background-image: url("../img/first-background.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  height: 35rem;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+}
+
+.content {
+  min-width: 35rem;
+  margin: auto;
+  color: azure;
+  padding-bottom: 3.5rem;
+}
+
+.content h2 {
+  font-weight: 300;
+  font-size: 2.7rem;
+  margin-bottom: 0;
+}
+.content p {
+  font-size: 1.6rem;
+  font-weight: 300;
+  margin-top: 1.2rem;
+}
+.content .learn-more {
+  color: azure;
+  background-color: #e0633a;
+  border: none;
+  border-radius: 4px;
+  padding: 8px 10px;
+  margin-top: 2.8rem;
+  font-size: 0.9rem;
+  font-family: sans-serif;
+  font-weight: 500;
+}
+
+.content .learn-more:hover {
+  color: #e0633a;
+  background-color: azure;
+}
+
+.products {
+  margin: 4rem 0;
+  text-align: center;
+}
+
+.product-cards {
+  display: flex;
+  justify-content: space-around;
+  margin: 0 5.5rem;
+}
+
+.products h2 {
+  font-family: "Montserrat", sans-serif;
+  font-size: 2.2rem;
+  padding: 1rem;
+}
+
+.products p {
+  font-size: 1.2rem;
+  padding: 1.1rem 0;
+}
+
+.cards {
+  padding: 1.3rem 0;
+}
+
+.card-logos {
+  width: auto;
+  height: 90px;
+  padding-left: 1.5rem;
+}
+
+.quoto-section {
+  display: flex;
+  height: 350px;
+  align-items: center;
+  margin: 25px 0;
+}
+
+.quoto-image {
+  height: 100%;
+  width: auto;
+}
+
+.quoto-section .quoto-container {
+  width: 100%;
+  height: 350px;
+  background-color: rgba(224, 99, 58, 0.1);
+  padding: 0 4.5rem;
+}
+
+.quoto-content {
+  margin: auto;
+  padding-top: 2.5rem;
+  display: block;
+}
+.quoto-content p {
+  margin-top: 1.8rem;
+  font-style: italic;
+  font-size: 2.2rem;
+}
+
+.quoto-content span {
+  color: #e0633a;
+  padding: 0 5px;
+}
+
+.get-karma {
```

Comment:
This is a main button which may be re-used elsewhere, so perhaps don't name it by its text content. 
We often refer to buttons as `.button--primary` or `.button--secondary` so we can create the buttons once and then re-use them throughout the site/app (this is following the [BEM notation](https://css-tricks.com/bem-101/) if you're wondering about the double dash)